### PR TITLE
make it more clear what launch-agent on kubernetes is

### PR DESCRIPTION
**Pull request by sebastian-lerner**
once we re-brand to "machine runner", this will read a lot nicer

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by devinmarieb** [comment]
@sebastian-lerner Can you double check that this is saying the same thing you were trying to say? That first sentence was a little hard to follow :)

> `launch-agent` (the component in charge of polling CircleCI for work) can be installed on Kubernetes and is the equivalent to container-agent for jobs that use the machine executor. `launch-agent` runs `task-agent` (the component in charge of executing work) within the same pod, as though it is running on a VM.

I'm also assuming we don't want to use `container-agent` vs "container agent"?

**Comment by sebastian-lerner** [comment]
i actually think we should just link to the runner concepts where we define launch agent.  instead of defining it in parens.  i didnt do it like that because adding links is always hard for me to do
